src/document/DocumentAnnotator.ts

```diff
@@ -176,6 +181,10 @@ export default class DocumentAnnotator extends BaseAnnotator {
             scrollToLocation(this.annotatedEl, annotationPageEl, {
                 offsets: centerHighlight(annotation.target.shapes),
             });
+        } else if (isDrawing(annotation)) {
+            scrollToLocation(this.annotatedEl, annotationPageEl, {
+                offsets: centerDrawing(annotation.target.path_groups),
```

Comment:
It looks like the only difference between the types is the offsets. Should we move the rest of the conditional logic to the bottom of the function?